core/src/test/kotlin/org/jetbrains/kotlinx/dataframe/api/constructors.kt

```diff
@@ -87,6 +87,7 @@ class ConstructorsTests {
         col.type() shouldBe typeOf<AnyRow>()
         col.kind() shouldBe ColumnKind.Group
         col[0] shouldBe row
+        @Suppress("DEPRECATION_ERROR")
```

Comment:
If we deprecate them, we should probably remove any internal uses

Reply:
good idea! But we should put somewhere that we will make it internal maybe. Or should we use this deprecation suppression mechanism see as "we're planning not to remove it but to make it internal"?